src/App/ImportUshahidiV2/Jobs/CreateDefaultSurvey.php

```diff
@@ -0,0 +1,191 @@
+            'options' => [],
+            'cardinality' => 1,
+        ],
+        [
+            'key' => 'location_name',
+            // 'original_id' => 'location_name',
+            'label' => 'Location Name',
+            'required' => 0,
+            'priority' => 0,
+            'default' => 0,
+            'type' => 'varchar',
+            'input' => 'text',
+            'options' => [],
+            'cardinality' => 1,
+        ],
+        [
+            'key' => 'location',
+            // 'original_id' => 'location',
+            'label' => 'Location',
+            'required' => 0,
+            'priority' => 0,
+            'default' => 0,
+            'type' => 'point',
+            'input' => 'location',
+            'options' => [],
+            'cardinality' => 1,
+        ],
+        [
+            'key' => 'verified',
+            // 'original_id' => 'verified',
+            'label' => 'Verified',
+            'required' => 0,
+            'priority' => 0,
+            'default' => 0,
+            'type' => 'int',
+            'input' => 'checkbox',
+            'options' => [],
+            'cardinality' => 1,
+        ],
+        [
+            'key' => 'news_source_link',
+            // 'original_id' => 'news',
+            'label' => 'News Source Link',
+            'required' => 0,
+            'priority' => 0,
+            'default' => 0,
+            'type' => 'link',
+            'input' => 'text',
+            'options' => [],
+            'cardinality' => 0,
+        ],
+        [
+            'key' => 'video_link',
+            // 'original_id' => 'news',
+            'label' => 'External Video Link',
+            'required' => 0,
+            'priority' => 0,
+            'default' => 0,
+            'type' => 'varchar',
+            'input' => 'video',
+            'options' => [],
+            'cardinality' => 0,
+        ],
+        [
+            'key' => 'photos',
+            // 'original_id' => 'news',
+            'label' => 'Photos',
+            'required' => 0,
+            'priority' => 0,
+            'default' => 0,
+            'type' => 'media',
+            'input' => 'upload',
+            'options' => [],
+            'cardinality' => 0,
+        ],
+        [
+            'key' => 'categories',
+            // 'original_id' => 'news',
+            'label' => 'Photos',
+            'required' => 0,
+            'priority' => 0,
+            'default' => 0,
+            'type' => 'tags',
+            'input' => 'tags',
+            'options' => [],
+            'cardinality' => 0,
+        ],
+    ];
+
+    /**
+     * Execute the job.
+     *
+     * @return void
+     */
+    public function handle(
+        ImportUshahidiV2\ImportMappingRepositoryContract $mappingRepo,
+        Entity\FormRepository $formRepo,
+        Entity\FormStageRepository $stageRepo,
+        Entity\FormAttributeRepository $attrRepo
+    ) {
+        // Create form
+        $formId = $formRepo->create(new Entity\Form([
```

Comment:
did we only have 1 survey in v2? (with no way to have more? )

Reply:
1 default, there is a way to add more fields. I'll do that in a follow up PR :)